### ISSUE TITLE
Expand built-in services

### ISSUE THREAD

**Issue by TonyRice** [storyscript/runtime#601]
I believe we should expand our current built-in services. A few ideas would be the following

crypto service (basic operations)
base64 service (encode, decode)
app service (exit, restart, info ,etc) - could be useful for graceful shutdowns!
pubsub service (pubsub in between your apps)

**Comment by wilzbach** [comment]
I strongly disagree! On the contrary we win if there are no built-ins as every built-in means more maintainance pressure and shows that out abstraction failed. There are more points like the difficulty to debug them or that if we need to introduce built-in services for everything the entire idea of a service hub doesn't work.
We need to eat our dog food here and look at why those can't be normal services and fix this instead. After all, we already have a base64 service: https://hub.storyscript.io/service/base64 and a hashing service: https://hub.storyscript.io/r/omg-services/hashes

The only service that I agree needs to be built-in for now is the `app` service to ensure only the main story has access to these commands.
